tests/functional-tests/src/test/groovy/pages/app/MinesOperatorPage.groovy

```diff
@@ -1,119 +1,170 @@
+      trainingEtiquette { $("input", type:"checkbox", "data-test":"cb-form-trainingEtiquette",dataState).parentsUntil("div", class:"v-input__slot") }
+      trainingLocations { $("input", type:"checkbox", "data-test":"cb-form-trainingLocations",dataState).parentsUntil("div", class:"v-input__slot") }
+      trainingFirstAid { $("input", type:"checkbox", "data-test":"cb-form-trainingFirstAid",dataState).parentsUntil("div", class:"v-input__slot") }
+      trainingReporting { $("input", type:"checkbox", "data-test":"cb-form-trainingReporting",dataState).parentsUntil("div", class:"v-input__slot") }
+
+      //Meals Preparation: Practice safe food handling
+      mealsDistancing { $("input", type:"checkbox", "data-test":"cb-form-mealsDistancing",dataState).parentsUntil("div", class:"v-input__slot") }
+      mealsDishware { $("input", type:"checkbox", "data-test":"cb-form-mealsDishware",dataState).parentsUntil("div", class:"v-input__slot") }
+      mealsDishwashing { $("input", type:"checkbox", "data-test":"cb-form-mealsDishwashing",dataState).parentsUntil("div", class:"v-input__slot") }
+
+      //Plan to manage individuals with suspected COVID-19 Infection
+      infectionSeparation { $("input", type:"checkbox", "data-test":"cb-form-infectionSeparation",dataState).parentsUntil("div", class:"v-input__slot") }
+      infectionSymptoms { $("input", type:"checkbox", "data-test":"cb-form-infectionSymptoms",dataState).parentsUntil("div", class:"v-input__slot") }
+      infectionHeathLinkBC { $("input", type:"checkbox", "data-test":"cb-form-infectionHeathLinkBC",dataState).parentsUntil("div", class:"v-input__slot") }
+      infectionSanitization { $("input", type:"checkbox", "data-test":"cb-form-infectionSanitization",dataState).parentsUntil("div", class:"v-input__slot") }
+      infectionAccommodation { $("input", type:"checkbox", "data-test":"cb-form-infectionAccommodation",dataState).parentsUntil("div", class:"v-input__slot") }
+
+      //Providing Food for Ill Workers
+      infectedFeeding { $("input", type:"checkbox", "data-test":"cb-form-infectedFeeding",dataState).parentsUntil("div", class:"v-input__slot") }
+
+      //Housekeeping for Ill Workers
+      infectedHousekeeping { $("input", type:"checkbox", "data-test":"cb-form-infectedHousekeeping",dataState).parentsUntil("div", class:"v-input__slot") }
+
+      //Waste Management for Ill Workers
+      infectedWaste { $("input", type:"checkbox", "data-test":"cb-form-infectedWaste",dataState).parentsUntil("div", class:"v-input__slot") }
+
+      certifyAccurateInformation { $("input", type:"checkbox", "data-test":"cb-form-certifyAccurateInformation",dataState).parentsUntil("div", class:"v-input__slot") }
+      agreeToInspection { $("input", type:"checkbox", "data-test":"cb-form-agreeToInspection",dataState).parentsUntil("div", class:"v-input__slot") }
+
+      subMit { $("button", class: "v-btn v-btn--contained theme--light v-size--default primary", "data-test":"btn-form-submit") }
```

Comment:
Guessing you didn't mean to capitalize the m in submit

Reply:
Needs a capital, but created another identifier